serialization/src/main/java/com/twitter/serial/util/InternalSerialUtils.java

```diff
@@ -114,13 +184,11 @@ public static int hashCode(@Nullable Object value, @NotNull Object... values) {
         return cast(TrivialComparator.INSTANCE);
     }
 
-    @SuppressWarnings("BlacklistedInterface")
     private static class TrivialComparator<T> implements Comparator<T>, Serializable {
```

Comment:
I don't think we use these comparators?

Reply:
They're used in CoreSerializers#getComparatorSerializer, though we might be able to remove that if it's not useful

Reply:
It looks like we're only using them in tests? Let's just get rid of them.